app/jobs/resolution_proofing_job.rb

```diff
@@ -14,7 +14,7 @@ class ResolutionProofingJob < ApplicationJob
   )
 
   def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
-              dob_year_only:, document_expired:)
+              dob_year_only:, document_expired:, flow_path: nil)
```

Comment:
Do we need to be handling the `flow_path` for the resolution job? Or are we just concerned about document capture?

And if yes:

- Should we be putting it into an event payload somewhere?
- How do we want to deal with [this other instance of it](https://github.com/18F/identity-idp/blob/ab26417f309f7962cdf3578ff84bab0d9d867bce/app/controllers/idv/gpo_controller.rb#L208-L213) that can be enqueued later in the flow? (extra context at LG-5122)

Reply:
No! I guess I got a bit too aggressive adding flow_path everywhere. Removed in 95e7cee